ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -8,6 +8,7 @@
         <include pattern="/etc/acs-commons/automatic-package-replication/jcr:content"/>
         <include pattern="/etc/acs-commons/bulk-workflow-manager"/>
         <include pattern="/etc/acs-commons/bulk-workflow-manager/jcr:content"/>
+        <include pattern="/etc/acs-commons/bulk-workflow-manager/rep:policy(/.*)?"/>
```

Comment:
Are we really failing to deploy a ton of `rep:policy` nodes?  I would imagine a ton of things are broken for new installs if that's the case, right?  Was this caused by recent changes?

Reply:
There is a bug in FileVault which might install the rep:policy nodes anyways, but as it has been clarified in https://issues.apache.org/jira/browse/JCRVLT-372 that would rather be a bug. Therefore I took the opportunity to clean things up here.

Reply:
Should be fixed now.